de.fu_berlin.inf.dpp.core/src/de/fu_berlin/inf/dpp/session/internal/SarosSession.java

```diff
@@ -1107,7 +1109,8 @@ private SarosSession(final String id, IContainerContext context,
                 .getInt(ColorNegotiationHook.KEY_INITIAL_COLOR);
             int hostFavoriteColorID = hostProperties
                 .getInt(ColorNegotiationHook.KEY_FAV_COLOR);
-            hostUser = new User(host, true, false, hostColorID,
+            boolean hostIsServer = hostProperties.getBoolean(ServerStatusHook.KEY_IS_SERVER);
+            hostUser = new User(host, true, false, hostIsServer, hostColorID,
                 hostFavoriteColorID);
             hostUser.setInSession(true);
             participants.put(hostUser.getJID(), hostUser);
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this use of "getJID"; it is deprecated. ([squid:CallToDeprecatedMethod](https://sonarcloud.io/organizations/saros/rules#rule_key=squid%3ACallToDeprecatedMethod))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=saros_complete&pullRequest=284&issues=AWc2MMoQ0EcSutaxS9dZ&open=AWc2MMoQ0EcSutaxS9dZ)

[ISSUE_KEY]: AWc2MMoQ0EcSutaxS9dZ